### DISCUSSION TITLE
希望在自定义主题中添加一个新的功能【启动当前选择的版本】

### DISCUSSION THREAD

**Discussion by WSXYT** [Meloong-Git/PCL#3965] [提问] [answered]
希望在自定义主题中添加一个新的功能【启动当前选择的版本】

**Comment by WForst-Breeze** [comment]
是自定义主页吧？

既然都在主页了为什么不直接按启动游戏呢🤔有点多此一举吧

**Reply by WSXYT** [reply]
主要是启动对应的服务器……

**Reply by WSXYT** [reply]
点击主页的按钮就可以一键启动当前版本的所选服务器了

**Reply by Deep-Dark-Forest** [reply]
~~？我不确定 PCL2 有没有一个功能是启动游戏后自动进入服务器~~
？关于我们群两位小朋友都在说高版本不生效这件事

**Reply by WForst-Breeze** [reply]
版本设置 - 设置 - 服务器 - 自动加入服务器 可以实现自动进入服务器

如果你想在自定义主页实现的话倒是也可以交个反馈

**Reply by Deep-Dark-Forest** [reply]
> 版本设置 - 设置 - 服务器 - 自动加入服务器 可以实现自动进入服务器
> 
> 如果你想在自定义主页实现的话倒是也可以交个反馈

好像仅对低版本生效（

**Reply by WForst-Breeze** [reply]
> 好像仅对低版本生效（

高版本试过了，有效

**Reply by MoYuan-CN** [reply]
高版本使用 `$quickplaypath` 文件做到，也不知道 Mojang 什么时候做了这么个东西

**Reply by WSXYT** [reply]
> ~？我不确定 PCL2 有没有一个功能是启动游戏后自动进入服务器~ ？关于我们群两位小朋友都在说高版本不生效这件事

有的

**Reply by WSXYT** [reply]
> 版本设置 - 设置 - 服务器 - 自动加入服务器 可以实现自动进入服务器
> 
> 如果你想在自定义主页实现的话倒是也可以交个反馈

我这个反馈就是为了这个……

**Reply by WForst-Breeze** [reply, marked as answer]
> 我这个反馈就是为了这个……

去 Issue 界面交反馈，感谢

**Reply by WorldHim** [reply]
#3997 (#4002)
